webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx

```diff
@@ -79,6 +89,16 @@ export class PlayerComparePlayStyleCharts extends React.PureComponent<Props, Sta
                         </Grid>
                     )
                 })}
+                <Grid container xs={12} justify="center" alignItems="center">
+                    <Grid item xs={12} style={{textAlign: "center"}}>
+                        {checkbox}
+                    </Grid>
+                    <Grid item xs={"auto"}>
```

Comment:
string props don't need to be in {}
```suggestion
                    <Grid item xs="auto">
```